### DISCUSSION TITLE
What is the purpose of the `toAlbum` parameter in the `Gal.requestAccess` function?

### DISCUSSION THREAD

**Discussion by alexislg2** [natsuk4ze/gal#187] [Q&A] [answered]
I could not find anything related to this parameter in the documentation

**Comment by natsuk4ze** [comment, marked as answer]
Hi @alexislg2 👋

### Saving to an album may require [additional permissions](https://github.com/natsuk4ze/gal/wiki/Permissions) compared to those that do not.
Therefore, set `toAlbum` to `true` when specifying `album` in `put-`.

https://github.com/natsuk4ze/gal/blob/16ebc2b03ba14f9002aa5d0801a17a7a356bb4dc/lib/src/gal.dart#L64

### Finally, don't forget to "Mark as answer ✅". Thank you.
